### PR TITLE
[Bebop] Support aggregate order

### PR DESCRIPTION
**Pull request by chrisngyn**

### CODE REVIEW

**Review by hiepnv90 — APPROVED** [review]
LGTM! :+1:

**Review by thanhpp — APPROVED** [review]

File: pkg/liquidity-source/bebop/rfq.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package bebop
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
 	"math/big"
```
Comment (NgoKimPhu) [inline pkg/liquidity-source/bebop/rfq.go:8]:
```suggestion
	"fmt"
	"math/big"

	"github.com/goccy/go-json"

```

File: pkg/liquidity-source/bebop/type.go [hunk 2]
```diff
@@ -1,5 +1,7 @@
 package bebop
 
+import "encoding/json"
```
Comment (NgoKimPhu) [inline pkg/liquidity-source/bebop/type.go:3]:
```suggestion
import (
	"github.com/goccy/go-json"
)
```

File: pkg/liquidity-source/bebop/type.go [hunk 3]
```diff
@@ -93,7 +102,16 @@ type QuoteSingleOrderResult struct {
...
 	} `json:"tx"`
-	ToSign struct { // the toSign part uses snake_case
+	ToSign            json.RawMessage `json:"toSign"`
+	OnchainOrderType  string          `json:"onchainOrderType"`
+	PartialFillOffset int             `json:"partialFillOffset"`
+}
+
+// ToSign struct will depend on the OnchainOrderType field of the QuoteSingleOrderResult.
+// Although, bebop support get quote for one to many and many to one, but we will only use one to one for now.
+// So, we will only define the ToSign struct for one to one.
```
Comment (NgoKimPhu) [inline pkg/liquidity-source/bebop/type.go:112]: is this comment still correct?

File: pkg/liquidity-source/bebop/rfq.go (hunk 1 above)
Comment (chrisngyn) [reply pkg/liquidity-source/bebop/rfq.go]: Updated

File: pkg/liquidity-source/bebop/type.go (hunk 2 above)
Comment (chrisngyn) [reply pkg/liquidity-source/bebop/type.go]: Updated

**Review by NgoKimPhu — APPROVED** [review]

**Pull request merged**